docs/_static/custom.css

```
@@ -1,7 +1,125 @@
-/* Set color of python role cross references to same as color for links */
+/* pylake logo colors */
+:root {
+  --gray: #404041;
+  --mediumgray: #666667;
+  --lightgray: #d9d9d9;
+  --lightestgray: #f0f0f0;
+  --pink: #f08d97;
+  --green: #84c175;
+  --blue: #1dabe3;
+  --orange: #f8ae4b;
+  --darkpink: #c07179;
+  --darkgreen: #6a9a5e;
+  --darkblue: #1789b6;
+  --darkorange: #c68b3c;
+  --lightpink: #f9d1d5;
+  --lightgreen: #cee6c8;
+  --lightblue: #a5ddf4;
+  --lightorange: #fcdfb7;
+}
+
+/* Links */
+a {
+  color: var(--blue);
+}
+
+a:visited {
+  color: var(--darkblue);
+}
+
+.wy-menu a {
+  color: var(--lightgray);
+}
+
+/* Cross references to same as color for links */
 code.xref.py {
-  color: #2980b9 !important;
+  color: var(--blue) !important;
 }
+
 code.xref.any.py {
-  color: #2980b9 !important;
+  color: var(--blue) !important;
+}
+
+/* Navigation side bar */
+.wy-side-nav-search {
+  background-color: var(--blue);
+}
+
+.wy-side-nav-search input[type=text] {
+  border-color: var(--darkblue);
+  border-width: 1.5px;
+}
+
+.wy-menu-vertical header, .wy-menu-vertical p.caption {
+  color: var(--blue);
+}
+
+/* Navigation side bar - version label */
+.wy-side-nav-search>div.version {
+  color: rgba(255,255,255,0.6);
+}
+
+/* API docs, blocks */
+.rst-content dl:not(.docutils) dt {
+  border-top: solid 5px var(--darkblue);
+  background-color: var(--lightblue);
+  width: 100%;
+  border-radius: 3px;
+}
+
+html.writer-html5 .rst-content dl[class]:not(.option-list):not(.field-list):not(.footnote):not(.glossary):not(.simple)>dt {
+  color: var(--darkblue);
+}
+
+.rst-content dl:not(.docutils) dl dt {
+  border-left: solid 3px var(--lightgray);
+  border-top: 0px;
+  background-color: var(--lightestgray);
+  padding-right: 25px;
+}
+
+/* inline code */
+.rst-content code.literal {
+  color: var(--gray);
+  border-color: var(--lightgray);
+  background-color: #edf0f2;
+  border-radius: 3px;
+}
+
+.rst-content code.download {
+  color: var(--blue);
+  background-color: rgba(0, 0, 0, 0);
+}
+
+/* code block, outer wrapper */
+.rst-content pre.literal-block, .rst-content div[class^='highlight'] {
+  border-radius: 3px;
+}
+
+/* note box */
+.rst-content .note .admonition-title {
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+  background-color: var(--blue);
+}
+
+.rst-content .note {
+  border-radius: 3px;
+  background-color: var(--lightblue);
+}
+
+/* warning box */
+.rst-content .warning .admonition-title {
+  border-top-right-radius: 3px;
+  border-top-left-radius: 3px;
+  background-color: var(--orange);
+}
+
+.rst-content .warning {
+  border-radius: 3px;
+  background-color: var(--lightorange);
+}
+
+.rst-content .warning a {
+  color: var(--orange);
```

I would make the text of links in a warning box darker, as I find the difference of the text of the link and the background to be uncomfortably small. I might have less rod cells ;-)
```suggestion
  color: var(--darkorange);
```

---

Yeah, I also find the links in those boxes hard to read.